src/pages/ApplyPage.tsx

```diff
@@ -154,6 +154,18 @@ function ApplyPage() {
             <li>Additional team meetings may be arranged by your Project Manager when necessary</li>
             "
           />
+          <Accordion
+            title="How are projects directed?"
+            active={active}
+            setActive={setActive}
+            description="
+                The process of directing a projects either starts:
+                </br>
+                <li>Focused: Project managers can choose to focus their project on a specific idea or concept from the get-go. These ideas are pitched at the first Tech Start meeting and members may then decide to join that team if they are interested! The teams then brainstorm adaptations and improvements to the idea, alongside the technologies and platforms to work with!</li>
```

Comment:
Maybe rephrase the answer to this question to be more tailored to general members, then have an answer more specific to PMs in a PM / Executive Team-specific FAQ section

Reply:
Completed